resources/assets/less/bem/mp-history-events.less

```diff
@@ -0,0 +1,59 @@
+/**
+ *    Copyright 2016 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+.mp-history-events {
+  .event() {
+    background-color: @grey-e;
+    max-width: 640px;
+
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .event-end() {
+    border-radius: 0 0 @border-radius-base @border-radius-base;
+    padding-bottom: @spacing * 2;
```

Comment:
since the rows already have padding-bottom of `(@spacing / 2)`, this should be `(@spacing * 2 - @spacing / 2)`.